modules/core/src/main/java/org/apache/ignite/internal/managers/discovery/ConsistentIdMapper.java

```diff
@@ -43,6 +44,21 @@ public ConsistentIdMapper(GridDiscoveryManager discoveryManager) {
     }
 
     /**
+     * Map topology version to list of consistent id.
+     *
+     * @param topVer Topology version.
+     * @return List of consistent ids.
+     */
+    public List<Object> mapToConsistentId(AffinityTopologyVersion topVer) {
```

Comment:
We should not calculate this list on-demand. Since the list is fixed on topologyVersion, this list should be moved to the DiscoCache